aip/general/0133.md

```diff
@@ -60,8 +60,8 @@ rpc CreateBook(CreateBookRequest) returns (Book) {
 - There **should** be exactly one `google.api.method_signature` annotation,
   with a value of `"parent,{resource},{resource}_id"`, or "`"parent,{resource}"`
   if the resource ID is not required.
-- If the API is operating on the [management plane][], the operation should have
-  [strong consistency][]: the completion of a create operation **must** mean
+- If the API is operating on the [management plane](https://google.aip.dev/111#management-plane), the operation should have
+  [strong consistency](https://google.aip.dev/121#strong-consistency): the completion of a create operation **must** mean
```

Comment:
We prefer to use text references instead of inline links. The reason this isn't linking properly is because the text reference at the bottom is not aligned with the text here. The text here is `strong consistency` but the text reference on line 216 (incorrectly) expects `strongly consistent`. That should be corrected to `strong consistency` instead so that this text reference link works unchanged.
```suggestion
  [strong consistency][]: the completion of a create operation **must** mean
```